Sources/GryphonLib/KotlinTranslator.swift

```diff
@@ -1262,12 +1262,21 @@ public class KotlinTranslator {
 		let indentation2 = increaseIndentation(indentation1)
 		if let getter = variableDeclaration.getter {
 			if let statements = getter.statements {
-				result.append(indentation1 + "get() {\n")
-				result.append(try translateSubtrees(
-					statements,
-					withIndentation: indentation2,
-					limitForAddingNewlines: 3))
-				result.append(indentation1 + "}\n")
+				result.append("\(indentation1)get() ")
+
+				if let singleExpressionStatement = isSingleExpressionFunction(statements, returnTypeString: getter.returnType) {
+					result.append("= ")
+					result.append(try translateExpression(singleExpressionStatement.expression,
+														  withIndentation: indentation1))
```

Comment:
Could you add a test that uses this indentation? Maybe a long function like before, or a closure with a statement in it if you prefer.

Reply:
I added it in #82. Needed to get some sleep last night. 😉